src/cpu/aarch64/jit_uni_reorder.hpp

```diff
@@ -1,6 +1,7 @@
 /*******************************************************************************
-* Copyright 2018-2020 Intel Corporation
-* Copyright 2020 FUJITSU LIMITED
+* Copyright 2018-2022 Intel Corporation
+* Copyright 2020-2022 FUJITSU LIMITED
+* Copyright 2022 Arm Ltd. and affiliates
```

Comment:
Hi Kawakami,

Thanks for your PR!

I see that you have included the Arm Licence header here and it jit_reorder.cpp.  I wanted to check if it is because my work towards the reorder implementation in this PR: https://github.com/oneapi-src/oneDNN/pull/1475 and if yes, I think it should be included as a separate commit and also list me as co-author?

Reply:
@cfRod 
Thank you for the comment and sorry for dropping out your commit log! 
I rebuilt the commit log of this PR. 

![image](https://user-images.githubusercontent.com/27798870/199134501-d9e28db3-7a93-4edb-9400-d262c1857c1e.png)

Reply:
Thanks!